aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1341,15 +1342,11 @@ Tensor coo_to_sparse_csc(const Tensor& self) {
 }
 
 Tensor coo_to_sparse_bsr(const Tensor& self, IntArrayRef blocksize) {
-  AT_ERROR(
-      "Conversion from ", self.layout(), " to SparseBsr is currently not supported.");
-  return self;
+  return self.to_sparse_csr().to_sparse_bsr(blocksize);
 }
 
 Tensor coo_to_sparse_bsc(const Tensor& self, IntArrayRef blocksize) {
-  AT_ERROR(
-      "Conversion from ", self.layout(), " to SparseBsc is currently not supported.");
-  return self;
+  return self.to_sparse_bsr(blocksize).to_sparse_bsc(blocksize);
```

Comment:
@pearu , could you please tell why there is a double conversion here? `to_sparse_bsc` does seem to have a COO dispatch...

Reply:
@nikitaved `self` here is a COO tensor and to convert it to BSC, we'll use an intermediate BSR tensor because we don't have direct COO->BSC implemented yet.

The COO dispatch in `to_sparse_bsc` is exactly this function here.

Reply:
Got it, thanks! Yet another thing that could be improved :)